fastlane/Fastfile

```
@@ -0,0 +1,119 @@
+def latest_googleplay_version_code(package_name)
+#   productionVersionCodes = google_play_track_version_codes(track: 'production', package_name: package_name)
+#   betaVersionCodes = google_play_track_version_codes(track: 'beta', package_name: package_name)
+#   alphaVersionCodes = google_play_track_version_codes(track: 'alpha', package_name: package_name)
+  internalVersionCodes = google_play_track_version_codes(track: 'internal', package_name: package_name)
+
+  # puts version codes from all tracks into the same array
+  versionCodes = [
+#     productionVersionCodes,
+#     betaVersionCodes,
+#     alphaVersionCodes,
+    internalVersionCodes
```

rubocop (1.69.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)